selfdrive/camerad/cameras/camera_common.cc

```diff
@@ -64,13 +62,13 @@ void CameraBuf::init(cl_device_id device_id, cl_context context, CameraState *s,
 
   rgb_width = ci->frame_width;
   rgb_height = ci->frame_height;
-#ifndef QCOM2
-  // debayering does a 2x downscale
-  if (ci->bayer) {
+
+  if (Hardware::EON() && ci->bayer) {
```

Comment:
```suggestion
  if (!Hardware::TICI() && ci->bayer) {
```

Should be the same, but consistent with choosing the CL file that actually chooses between 2x downscale.